spec.bs

```diff
@@ -111,10 +111,17 @@ interface DocumentPictureInPicture : EventTarget {
   attribute EventHandler onenter;
 };
 
+enum DocumentPictureInPictureColorMode {
+  "preferDark",
+  "preferLight",
+  "preferDevice"
```

Comment:
What do you think about "preferDefault"? Also, I feel like this should be the first entry in the enum